specs/paymentrequest.html

```diff
@@ -284,16 +284,77 @@
           payment app.
         </div>
         <div class="note">
-          <p>The <code>supportedMethods</code> sequence contains the <a>payment method identifiers</a>
-          for the <a>payment methods</a> that the merchant web site accepts.</p>
+
+          <p>The <code>details</code> contains the core payment request data that will be passed on to the user selected
+            <a>payment app</a>.</p>
+
+          <p>The <code>amount</code> sequence contains the amount (possibly in different currencies) that the website
```

Comment:
Is this "amount" in the top-level details object, or is it in every paymentMethod object? I'm a -1 for the former (top-level) and a +1 for the latter (in every payment method object). The -1 is because it feels like doing so would duplicate data (because we'd also have that same data in each payment method object?)... sorry, can't see the complete spec to see where the change is (and don't have the time to checkout this branch and view it locally). Hopefully this comment makes sense to you @adrianhopebailie.